### ISSUE TITLE
Secondary research on ReportStream competitor authentication functionality

### ISSUE THREAD

**Issue by dylansmith-Nava** [CDCgov/prime-reportstream#10281]
## User Story
As a ReportStream teammate, I want to understand the authentication functionality at ReportStream competitors, so we can have a better understanding of the overall landscape of user authentication in the ReportStream environment.

## Background and context
As part of the design brief to add additional third-party authentication providers and abstract our authentication layers, we will need to consider identity proofing as a separate functionality from Okta authentication. As more and more Senders are added to the Universal Pipeline in the near and far future, we will need to prove that these organizations 'are who they say they are', and are not bad actors that could send faulty or fraudulent data into ReportStream.

## Open questions
- What users authentication capabilities exist at AIMS?
- What users authentication capabilities exist at Rhapsody?

## Working links
- Part of [Authentication enhancements design brief](https://docs.google.com/document/d/1mHy9D9WH-pe1xf4mw6u6FKsojiK9vDdIdgTGnmdchDk/edit#)
- Discussion guide TBD

## Acceptance criteria
- [x] Documentation on authentication capability(s) at AIMS
- [x] Documentation on authentication capability(s) at Rhapsody

## Not in scope
- [] Talking directly to AIMS + Rhapsody
- [] Deciding what authentication capability(s) ReportStream should have

**Comment by dylansmith-Nava** [comment]
No progress made here yet; waiting on the Rhapsody demo and having difficulty learning more about AIMS.

**Comment by dylansmith-Nava** [comment]
Asked a question during the Rhapsody demo and learned a bit more about their identity proofing. Notes here: https://docs.google.com/document/d/1Pz-8KEj5ag0j1hHTwy6B-hqQKiFk_EOVdmhXn8wpG2E/edit#heading=h.5d3xmmoa5n8j

**Comment by dylansmith-Nava** [comment]
Briefly discussed AIMS identity proofing via Slack message between Chris K., Brick, and a colleague named Adrian. It does not appear that AIMS has any identity proofing capabilities beyond what ReportStream currently does.

**Comment by dylansmith-Nava** [comment]
As both Rhapsody and AIMS identity proofing capabilities have been documented, I am moving this ticket to done!
